src/Altinn.Apps/AppFrontend/react/receipt/src/features/receipt/containers/Receipt.tsx

```diff
@@ -136,13 +147,26 @@ function Receipt(props: WithStyles<typeof styles>) {
   }, [user]);
 
   return (
-    <>
+    <Grid
+      container={true}
+      direction='column'
+      justify='center'
+      alignItems='center'
+    >
       <AltinnAppHeader
         logoColor={theme.altinnPalette.primary.blueDarker}
         headerBackgroundColor={theme.altinnPalette.primary.blue}
         party={party || {} as IParty}
         userParty={user ? user.party : {} as IParty}
       />
+      {instance?.status?.substatus &&
+        <Grid item={true} className={props.classes.substatus}>
+          <AltinnSubstatusPaper
+            label={getTextResourceByKey(instance.status.substatus.label, textResources)}
+            description={getTextResourceByKey(instance.status.substatus.description, textResources)}
```

Comment:
do we need to check that there is a description before trying to retrieve the textResource or is there ok null handling in the `getTextResourceByKey`-method?

Reply:
Jupp, handled in `getTextResourceByKey`.